rfcs/2021-05-17-7469-load-balancing-aggregators.md

```diff
@@ -0,0 +1,99 @@
+# RFC 7469 - 2021-05-16 - Scaling and load balancing for Vector aggregators
+
+This RFC describes the need for a tooling and environment agnostic load balancing solution to be bundled with Vector aggregator deployments.
+
+* [Scope](#scope)
+* [Motivation](#motivation)
+* [Internal Proposal](#internal-proposal)
+* [Rationale](#rationale)
+* [Prior Art](#prior-art)
+* [Drawbacks](#drawbacks)
+* [Alternatives](#alternatives)
+  * [Do Nothing](#do-nothing)
+  * [Only Client-side Load Balancing](#only-client-side-load-balancing)
+  * [Require a Service Mesh](#require-a-service-mesh)
+* [Outstanding Questions](#outstanding-questions)
+* [Plan Of Attack](#plan-of-attack)
+
+## Scope
```

Comment:
It seems to me the scope of this is also targeted specifically at running aggregators in k8s. Is that accurate? Or do we want to cover aggregator use-cases outside of k8s?

Reply:
I think I definitely was viewing it from kubernetes angle (probably due to my background) but I don't think it would be crazy to include steps to configure/install on other platforms - if we want. @binarylogic what do you think?

Reply:
I'm all for narrowing scope initially. Let's make this k8s specific to start. We can address other platforms later.